custom/scripts/push_images/README.md

```diff
@@ -0,0 +1,441 @@
+## Table of content
+- [Supported Tunables](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#supported-tunables)
+- [Check the available commands](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#check-the-available-commands)
+- [List Down all the Images](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#list-down-all-the-images)
+- [Pull the LitmusChaos Images into your machine](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#pull-the-litmuschaos-images-into-your-machine)
+- [Push the LitmusChaos Images to your image registry](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#push-the-litmuschaos-images-to-your-image-registry)
+
+## LitmusChaos Images
+
+- Pish docker images contains all the images that are used to execute a litmuschaos generic experiment using litmus portal. For more information please check [LitmusChaos Docs](https://litmusdocs-beta.netlify.app/docs/introduction).
+
+
+## Get LitmusChaos Images In Your Repository
+
+- For pulling the litmus image and pushing into your registry please follow the given steps:
+
+```bash
+wget https://raw.githubusercontent.com/litmuschaos/test-tools/master/custom/scripts/push_images/litmus_image_push.sh
+chmod +x litmus_image_push.sh
+```
+
+#### Supported Tunables
+
+ <table>
```

Comment:
Can we create a table to list all the portal_images, backend_images, and workflow_images inside README?